contracts/CertiKSecurityOracle.sol

```diff
@@ -27,11 +27,23 @@ contract CertiKSecurityOracle is Ownable {
   constructor() public {
     initialize();
   }
+  
+  function isContract(address _addr) private view returns (bool) {
```

Comment:
Perhaps we can also make it a public helper function?

Reply:
Updated.